parallel_wavegan/utils/utils.py

```diff
@@ -388,7 +396,7 @@ def download_pretrained_model(tag, download_dir=None):
                 for member in tar.getmembers():
                     if member.isreg():
                         member.name = os.path.basename(member.name)
-                        tar.extract(member, f"{download_dir}/{tag}")
-    checkpoint_path = find_files(f"{download_dir}/{tag}", "checkpoint*.pkl")
+                        tar.extract(member, f"{download_dir}/{tag_or_url}")
+    checkpoint_path = find_files(f"{download_dir}/{tag_or_url}", "checkpoint*.pkl")
```

Comment:
Maybe using url as the directory name looks redundant.
Why don't you use tag for tag specification and URL id for URL specification?
Maybe we can define tag for each case e.g., tag=tag_or_url and tag=id_

Reply:
I like your second suggestion, I define a tag for each case, it looks cleaner.